sdk/textanalytics/azure-ai-textanalytics/swagger/README.md

```diff
@@ -0,0 +1,90 @@
+# Text Analytics for Python
```

Comment:
hmmm could u expand more on what is this? is this so u can do customization on top of the original swagger?

Reply:
what changed that required this addition?

Reply:
@abhahn and I had discussed moving the readme we generate to the repo, kind of like how [js](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/textanalytics/ai-text-analytics/swagger) does it. Since we're doing all of these directives to override the generate pollers etc, we thought we would move it here, bc that is very python-sdk specific

Reply:
we do the same in .NET, I just didn't see anything about it in the description of the PR so it was a surprise and wanted to understand more